app/nav/CmsNavbar.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+	Flex,
+	Stack,
+	Button,
+	useColorModeValue,
+	Heading,
+} from "@chakra-ui/react";
+import { FiHome } from "react-icons/fi";
+import { Form } from "remix";
+import { url as logoutURL } from "~routes/cms.logout";
+import { url as cmsUrl } from "~routes/cms/index";
+import NavElement from "./CmsNavElement";
+
+export default function CmsNavbar({ fullName }: { fullName: string }) {
+	return (
+		<Flex
+			bg={useColorModeValue("gray.200", "gray.700")}
+			w="13%"
+			minW="min-content"
+			direction="column">
+			<Flex
+				borderBottom="2px"
+				borderColor={useColorModeValue("gray.300", "gray.600")}
+				w="100%"
+				pl={4}
+				alignItems="center">
+				<Heading my={6} size="md">
+					{fullName}
+				</Heading>
+			</Flex>
+			<Flex
```

Comment:
These nested `Flex` components have no semantic meaning. I think these should either be left out or replaced with native `chakra-ui` ones that serves their purpose.

Reply:
Yeah you're right. I copeid and pasted my previous implementation and forgot to remove some unnecessary parts